### DISCUSSION TITLE
Embed Styling - Title Bar Color

### DISCUSSION THREAD

**Discussion by blissiraDC** [FlowiseAI/Flowise#1511] [Q&A]
How to change the title bar colour from blue in the styling config? Can't see any option for it. Any help on this so much appreciated, its the last deal breaker! Thanks in advance

**Comment by HenryHengZJ** [comment]
@blissiraDC you can change the color via theme -> button -> backgroundColor:
```
<script type="module">
  import Chatbot from 'https://cdn.jsdelivr.net/npm/flowise-embed/dist/web.js';
  Chatbot.init({
    chatflowid: 'abc',
    apiHost: 'http://localhost:3000',
    theme: {
      button: {
        backgroundColor: '#3B81F6'
      }
   }
})
```

**Reply by googlepoke** [reply]
the css to change the background color of showTitle background is defaulted to BLUE unable to change it any ideas?
